gui/src/components/maintenance/cortx-resource.vue

```diff
@@ -114,21 +108,18 @@
                 class="cortx-btn-primary"
                 :disabled="!resource.shutdown"
                 @click="shutdownSelectedResource()"
-              >
-                {{ $t("systemMaintenance.apply-btn") }}
-              </button>
+              >{{ $t("systemMaintenance.apply-btn") }}</button>
             </v-col>
           </v-row>
           <div
             class="cortx-text-md mt-5 font-weight-bold"
             id="lblShutdownNode"
             v-if="shutdownNode"
-          >{{ $t('systemMaintenance.shutdown-note', { node: shutdownNode }) }}
-          </div>
```

Comment:
is this file contain only formatting changes?

Reply:
@namrata-khake-ui  @kupranay I have tried to fix all the linting issues with this ticket.